packages/botonic-dx/botonic-app-config/webpack/api-ws.js

```diff
@@ -21,10 +21,10 @@ module.exports = ({
   output: {
     path: path.resolve(projectPath, 'dist/websocket'),
     filename: 'server.js',
-    libraryTarget: 'commonjs',
+    libraryTarget: 'umd',
```

Comment:
Out of curiosity, what's the benefit or motivation of this change?

Reply:
The first fact is that we had an inconsistency between `api-ws.js` and `api-rest.js`. `api-rest.js`'s libraryTarget's was specifying the `umd` version which is the one that makes the project to be run as expected. Both api-ws and api-rest should be transpiled almost in the same way. Curiously, `commonjs` target, which is the one that targets NodeJS syntax, seems to be the logic one, but in pratice it is not working. `umd` ensures that the bundle works both in server-side and in client-side, so we are leaving as is in this iteration. We should do further investigation when optimizing bundle sizes for every artifact generated for Botonic 1.0.